SushiRunner/Views/Home/_Meal.cshtml

```diff
@@ -1,12 +1,15 @@
 @model MealModel
 
 <div class="item-cart" data-meal-id="@Model.Id">
-    <div class="item-img">
-        <img src="@Model.ImagePath" alt="@Model.Name">
-    </div>
-    <div class="item-title">
-        <h4>@Model.Name</h4>
-    </div>
+    <a id="@Model.Id" href="/Home/Meals/@Model.Id">
```

Comment:
Use [this](https://www.mikesdotnetting.com/article/274/introducing-taghelpers-in-asp-net-mvc-6) instead